btr-web/btr-main-app/components/individual-person/summary-table/Index.vue

```diff
@@ -193,77 +162,19 @@ function capFirstLetterInName (fullName: string) {
     </template>
 
     <template #table-row="{ item, index } : { item: SiSchemaType, index: number }">
-      <tr v-if="editingIndex != index" class="border-t-[1px] border-gray-400" data-cy="summary-table-row">
-        <td data-cy="summary-table-name">
-          <PersonInfoName
-            :badges="getBadges(item)"
-            :item="{
-              'legalName': item.name.fullName,
-              'alternateName': item.name.preferredName,
-              'birthDate': item.birthDate,
-              'class': getOpacityClass(item)}"
-          />
-          <PersonInfoCitizenship
-            :class="getOpacityClass(item)"
-            :nationalities="item.citizenships"
-          />
-        </td>
-        <td class="align-top" :class="getOpacityClass(item)" data-cy="summary-table-details">
-          <PersonInfoDetails :item="item" />
-        </td>
-
-        <td :class="getOpacityClass(item)" data-cy="summary-table-controls">
-          <PersonInfoControl :item="item" />
-        </td>
-
-        <td :class="getOpacityClass(item)" data-cy="summary-table-dates">
-          <p v-for="date in item.effectiveDates" :key="date.startDate">
-            {{
-              $t('texts.dateRange', {
-                start: date.startDate ? date.startDate : $t('labels.unknown'),
-                end: date.endDate ? date.endDate : $t('labels.current')
-              })
-            }}
-          </p>
-        </td>
-        <!-- NOTE: getOpacityClass is not applied to the edit column on purpose -->
-        <template v-if="edit">
-          <td data-cy="summary-table-buttons" class="action-button align-top">
-            <IndividualPersonSummaryTableActionButton
-              :button="getActionButton(item, index)"
-              :dropdown-items="getActionDropDownItems(item, index)"
-            />
-          </td>
-        </template>
-      </tr>
-
-      <!--
-          placeholder row for display warning message for minor SI
-          the v-if is set to false until the feature is implemented;
-          rules for minor: #20621
-        -->
-      <tr v-if="editingIndex != index && false" data-cy="summary-table-row-minor">
-        <td colspan="5">
-          TBD - Message for Minor SI
-        </td>
-      </tr>
-
-      <!-- to be updated in #21660 -->
-      <tr v-if="editingIndex != index && item.missingInfoReason !== ''" data-cy="summary-table-row-missing-info">
-        <td colspan="5">
-          TBD - Unable to obtain or confirm information
-          {{ item.missingInfoReason }}
-        </td>
-      </tr>
-
-      <tr v-if="editingIndex != index && item.ui.showCeaseDateInput" data-cy="summary-table-row-cease">
-        <td colspan="5">
-          <IndividualPersonSummaryTableCessationDateEntry :individual="item" :index="index" />
-        </td>
-      </tr>
-
+      <IndividualPersonSummaryTableRowUpdating
+        v-if="item.ui.updating && editingIndex != index"
+        :si="item"
+      />
```

Comment:
this puts loader animation over the problem areas of the row that nuxt/ui renders incorrectly after we change them (does not pick up changes to the si field arrays properly) -- previously was handling this with setting the whole list of SIs to empty and then setting them back causing a flicker (was not pretty)